### ISSUE TITLE
Three-stage training

### ISSUE THREAD

**Issue by Zhaosoi** [yayafengzi/LMM-HiMTok#2]
In the paper, the 3-stage training is metioned. In the github, I noted only the files for the second and third stage training are uploaded. How do I perform the first stage training? @yayafengzi

**Comment by ccx1997** [comment]
The 1st stage is conducted based on the TiTok code. Kindly refer to it.

**Comment by jntesai** [comment]
According to the paper the hierarchical mask loss is used during first stage training as well right? So it is not just feeding mask data to TiTok training. Could you share your training recipe?

**Comment by jntesai** [comment]
Also, TiTok stage 1 training relies on a pre-trained MaskGIT-VQGAN to generate proxy codes as learning targets. That doesn't seem relevant if our training target is binary mask. Did you skip the stage 1 training?
